war/src/main/java/io/lumeer/engine/controller/ProjectFacade.java

```diff
@@ -147,7 +148,7 @@ public Project readProject(final String projectCode) {
     *       project data
     * @return id of the organization
     */
-   public String createProject(final Project project) {
+   public String createProject(final Project project) throws InvalidDocumentKeyException {
       DataDocument dataDocument = project.toDataDocument()
                                          .append(LumeerConst.Project.ATTR_ORGANIZATION_ID, organizationFacade.getOrganizationId());
       String id = dataStorage.createDocument(LumeerConst.Project.COLLECTION_NAME, dataDocument);
```

Comment:
Same here. Move adding role to `databaseInitilizer.onOrganizationCreated` method and don't throw exception because we know, that we add correct role.